packages/components/src/Button/GenericButton/GenericButton.tsx

```diff
@@ -225,9 +225,9 @@ const renderCustomComponent = (
           props["aria-describedby"] === null
             ? undefined
             : classnames(
-              contextProps["aria-describedby"],
-              linkProps["aria-describedby"]
-            )
+                contextProps["aria-describedby"],
+                linkProps["aria-describedby"]
+              )
```

Comment:
These are just linting fixes no actual change from what was already on the tooltip branch